db/migrations/20220421221517-insert-association.ts

```diff
@@ -2,21 +2,45 @@ import {Sequelize_migration} from "../util/inteface";
 import {QueryInterface} from "sequelize";
 
 export const up = async (queryInterface: QueryInterface, Sequelize: Sequelize_migration) => {
-  await queryInterface.changeColumn('user', 'teams', {
-  type: Sequelize.INTEGER,
-    references: {
-      model: 'user-team',
-      key: 'id'
+  // await queryInterface.changeColumn('user', 'teams', {
+  // type: Sequelize.INTEGER,
+  //   references: {
+  //     model: 'user-team',
+  //     key: 'id'
+  //   },
+  // });
+
+  await queryInterface.addConstraint('user', {
+    fields: ['teams'],
+    type: 'foreign key',
+    name: 'custom_fkey_constraint_name',
+    references: { //Required field
+      table: 'user-team',
+      field: 'id'
     },
+    onDelete: 'cascade',
+    onUpdate: 'cascade'
   });
 
-  await queryInterface.changeColumn('team', 'users', {
-  type: Sequelize.INTEGER,
-    references: {
-      model: 'user-team',
-      key: 'id'
+  await queryInterface.addConstraint('team', {
+    fields: ['users'],
+    type: 'foreign key',
+    name: 'custom_fkey_constraint_name',
+    references: { //Required field
+      table: 'user-team',
+      field: 'id'
     },
+    onDelete: 'cascade',
+    onUpdate: 'cascade'
   });
+
+  // await queryInterface.changeColumn('team', 'users', {
```

Comment:
rm if not needed. also can you explain what's the differernce between the commented out piece of code and one above?

Reply:
_changeColumn_ - changes the column, as you can see, I re-specify its type and add a foreign key.
_addConstraint_ - in my case, only adds a foreign key to this column without changing its type